verify.go

```diff
@@ -17,53 +17,51 @@ import (
 // IDTokenVerifier provides verification for ID Tokens.
 type IDTokenVerifier struct {
 	keySet *remoteKeySet
-	config *verificationConfig
+	config *Config
+	issuer string
 }
 
-// verificationConfig is the unexported configuration for an IDTokenVerifier.
-//
-// Users interact with this struct using a VerificationOption.
-type verificationConfig struct {
-	issuer string
-	// If provided, this value must be in the ID Token audiences.
-	audience string
-	// If not nil, check the expiry of the id token.
-	checkExpiry func() time.Time
+// Config is the configuration for an IDTokenVerifier.
+type Config struct {
+	// If provided, ClientID must be in the ID Token audiences.
+	// It ensures that an ID Token was issued for the specific client.
+	ClientID string
+	// If provided, verify nonce.
+	ClaimNonce func(nonce string) error
 	// If specified, only these sets of algorithms may be used to sign the JWT.
-	requiredAlgs []string
-	// If not nil, don't verify nonce.
-	nonceSource NonceSource
-}
+	// Defaults to RS256.
```

Comment:
Can we expand this doc to include some of the existing signing algorithm comment? https://godoc.org/github.com/coreos/go-oidc#VerifySigningAlg